lib/osut/utils.rb

```diff
@@ -1684,6 +1684,80 @@ def offset(p1 = [], w = 0, v = 0)
     end
   end
 
+  ##
+  # Validate whether an OpenStudio planar surface is safe to process.
+  #
+  # @param surface [OpenStudio::Model::PlanarSurface] a surface
+  #
+  # @return [Bool] true if valid surface
+  def valid(surface = nil)
```

Comment:
A clone of TBD's [validate](https://github.com/rd2/tbd/blob/56f4daf61da39b35d849ee4bce224737b7ce684b/lib/tbd/geo.rb#L227) - to be deprecated within TBD (in favour of OSut's `valid?` ... note: add question mark).